### PR TITLE
reference/tools/tikv-control: add TiKV control in Chinese

### PR DESCRIPTION
**Pull request by dcalvin**
Per feedback in the Wechat group, updated and added the Chinese version of TiKV control instead of linking it to the external file in English.  @yikeke PTAL

Link to the English version: https://github.com/tikv/tikv/blob/master/docs/tools/tikv-control.md

### CODE REVIEW

File: dev/reference/tools/tikv-control.md [hunk 1]
```diff
@@ -1,29 +1,34 @@
 ---
 title: TiKV Control 使用说明
-category: reference
-aliases: ['/docs-cn/tools/tikv-control/']
+category: tools
```
Comment (dcalvin) [inline dev/reference/tools/tikv-control.md]:
```suggestion
category: reference
```

**Comment by dcalvin** [comment]
@hicqu PTAL. Thanks

File: dev/reference/tools/tikv-control.md [hunk 2]
```diff
@@ -164,24 +169,25 @@ DebugClient::check_region_consistency: RpcFailure(RpcStatus { status: Unknown, d
...
 
-### Dump snapshot meta
+### Dump snapshot 元信息
 
 这条子命令可以用于解析指定路径下的 Snapshot 元文件并打印结果。
 
 ### 打印 Raft 状态机出错的 Region
 
-前面 tombstone 命令可以将 Raft 状态机出错的 Region 设置为 Tombstone 状态，避免 TiKV 启动时对它们进行检查。在运行那个命令之前，`bad-regions` 命令可以找出这些出错了的 Region，以便将多个工具组合起来进行自动化的处理。
+前面 `tombstone` 命令可以将 Raft 状态机出错的 Region 设置为 Tombstone 状态，避免 TiKV 启动时对它们进行检查。在运行 `tombstone` 命令之前，可使用 `bad-regions` 命令可以找出出错了的 Region，以便将多个工具组合起来进行自动化的处理。
```
Comment (yikeke) [inline dev/reference/tools/tikv-control.md]:
```suggestion
前面 `tombstone` 命令可以将 Raft 状态机出错的 Region 设置为 Tombstone 状态，避免 TiKV 启动时对它们进行检查。在运行 `tombstone` 命令之前，可使用 `bad-regions` 命令找到出错的 Region，以便将多个工具组合起来进行自动化的处理。
```

File: dev/reference/tools/tikv-control.md [hunk 3]
```diff
@@ -164,24 +169,25 @@ DebugClient::check_region_consistency: RpcFailure(RpcStatus { status: Unknown, d
...
 
 ```bash
 $ tikv-ctl --db /path/to/tikv/db bad-regions
 all regions are healthy
 ```
 
-命令执行成功会打印上面的信息，否则会打印出有错误的 Region 列表。目前可以检出的错误包括 last index、commit index 和 apply index 之间的不匹配，以及 Raft log 的丢失。其他一些情况，比如 Snapshot 文件损坏等仍然需要后续的支持。
+命令执行成功会打印以上信息，否则会打印出有错误的 Region 列表。目前可以检出的错误包括 `last index`、`commit index` 和 `apply index` 之间的不匹配，以及 Raft log 的丢失。其他一些情况，比如 Snapshot 文件损坏等仍然需要后续的支持。
```
Comment (yikeke) [inline dev/reference/tools/tikv-control.md]:
```suggestion
命令执行成功后会打印以上信息，否则会打印出有错误的 Region 列表。目前可以检出的错误包括 `last index`、`commit index` 和 `apply index` 之间的不匹配，以及 Raft log 的丢失。其他一些情况，比如 Snapshot 文件损坏等仍然需要后续的支持。
```

File: dev/reference/tools/tikv-control.md [hunk 4]
```diff
@@ -164,24 +169,25 @@ DebugClient::check_region_consistency: RpcFailure(RpcStatus { status: Unknown, d
...
 
-### 查看 Region 的 properties 信息
+### 查看 Region 信息
```
Comment (yikeke) [inline dev/reference/tools/tikv-control.md]:
```suggestion
### 查看 Region 属性
```
If you agree that properties should be rendered as "属性", please update the same words in this section.

File: dev/reference/tools/tikv-control.md [hunk 5]
```diff
@@ -200,7 +206,7 @@ $ tikv-ctl --host 127.0.0.1:20160 region-properties -r 2
 使用 `modify-tikv-config` 命令可以动态修改配置参数，暂时仅支持对于 RocksDB 相关参数的动态更改。
 
 - `-m` 用于指定要修改的模块，有 `storage`、`kvdb` 和 `raftdb` 三个值可以选择。
-- `-n` 用于指定配置名。配置名可以参考 [TiKV 配置模版](https://github.com/pingcap/tikv/blob/master/etc/config-template.toml#L213-L500)中 `[storage]`、`[rocksdb]` 和 `[raftdb]` 下的参数，分别对应 `storage`、`kvdb` 和 `raftdb`。同时，还可以通过 `default|write|lock + . + 参数名` 的形式来指定的不同 CF 的配置（对于 `kvdb` 有 `default|write|lock` 可以选择，对于 `raftdb` 仅有 `default` 可以选择。
+- `-n` 用于指定配置名。配置名可以参考 [TiKV 配置模版](https://github.com/pingcap/tikv/blob/master/etc/config-template.toml#L213-L500)中 `[storage]`、`[rocksdb]` 和 `[raftdb]` 下的参数，分别对应 `storage`、`kvdb` 和 `raftdb`。同时，还可以通过 `default|write|lock + . + 参数名` 的形式来指定的不同 CF 的配置。对于 `kvdb` 有 `default`,`write`、`lock` 可以选择，对于 `raftdb` 仅有 `default` 可以选择。
```
Comment (yikeke) [inline dev/reference/tools/tikv-control.md]:
```suggestion
- `-n` 用于指定配置名。配置名可以参考 [TiKV 配置模版](https://github.com/pingcap/tikv/blob/master/etc/config-template.toml#L213-L500)中 `[storage]`、`[rocksdb]` 和 `[raftdb]` 下的参数，分别对应 `storage`、`kvdb` 和 `raftdb`。同时，还可以通过 `default|write|lock + . + 参数名` 的形式来指定的不同 CF 的配置。对于 `kvdb` 有 `default`、`write` 和 `lock` 可以选择，对于 `raftdb` 仅有 `default` 可以选择。
```

File: dev/reference/tools/tikv-control.md [hunk 6]
```diff
@@ -218,9 +224,9 @@ success!
 
 ### 强制 Region 从多副本失败状态恢复服务
 
-`unsafe-recover remove-fail-stores` 命令可以将一些失败掉的机器从指定 Region 的 peers 列表中移除。这个命令只有 local 模式，而且需要目标 TiKV 先停掉服务以便释放文件锁。
+`unsafe-recover remove-fail-stores` 命令可以将故障机器从指定 Region 的 peers 列表中移除。运行命令之前，需要目标 TiKV 先停掉服务以便释放文件锁。
```
Comment (yikeke) [inline dev/reference/tools/tikv-control.md]:
```suggestion
`unsafe-recover remove-fail-stores` 命令可以将故障机器从指定 Region 的 peer 列表中移除。运行命令之前，需要目标 TiKV 先停掉服务以便释放文件锁。
```

File: dev/reference/tools/tikv-control.md [hunk 7]
```diff
@@ -250,6 +256,30 @@ success!
...
+
+dump 现有 RocksDB 的声明：
+
+```bash
+$ tikv-ctl ldb --hex manifest_dump --path=/tmp/db/MANIFEST-000001
+```
+
+您可以通过 `--column_family=<string>` 指定查询的目标列族。
+
+通过 `--try_load_options` 命令加载数据库选项文件以打开数据库。数据库运行的时候，建议您保持该命令为开启的状态。如果您使用默认配置打开数据库，LSM-tree 存储组织可能会出现混乱，且无法自动恢复。
```
Comment (yikeke) [inline dev/reference/tools/tikv-control.md]:
```suggestion
通过 `--try_load_options` 命令加载数据库选项文件以打开数据库。在数据库运行时，建议您保持该命令为开启的状态。如果您使用默认配置打开数据库，LSM-tree 存储组织可能会出现混乱，且无法自动恢复。
```

File: dev/reference/tools/tikv-control.md [hunk 8]
```diff
@@ -250,6 +256,30 @@ success!
...
 > - `-p` 选项指定 PD 的 endpoint，不使用 `http` 前缀，用于查询指定的 `region_id` 是否有效。
-> - 对于指定 Region 的 peers 所在的每个 store，均须执行这个命令。
+> - 对于指定 Region 的 peers 所在的每个 store，均须执行该命令。
+
+
+### Ldb 命令
+
+ldb 命令行工具提供多种数据访问以及数据库管理命令。下方列出了一些示例用法。详细信息请在运行 `tikv-ctl ldb` 命令时查看帮助消息或查阅 RocksDB 文档。
+
+数据访问序列示例：
```
Comment (yikeke) [inline dev/reference/tools/tikv-control.md]:
```suggestion
数据访问序列示例如下。
```

File: dev/reference/tools/tikv-control.md (hunk 4 above)
Comment (dcalvin) [reply dev/reference/tools/tikv-control.md]: Sounds good.

**Pull request merged**